packages/store/src/symbols.ts

```diff
@@ -132,9 +135,12 @@ export interface StoreOptions<T> {
   children?: any[];
 }
 
-export const enum LifecycleHooks {
-  NgxsOnInit = 'ngxsOnInit',
-  NgxsAfterBootstrap = 'ngxsAfterBootstrap'
+/**
+ * Represents a basic change from a previous to a new value for a single state instance.
+ * Passed as a value in a NgxsSimpleChanges object to the ngxsOnChanges hook.
+ */
+export class NgxsSimpleChange<T = any> {
+  constructor(public previousValue: T, public currentValue: T, public firstChange: boolean) {}
```

Comment:
Just in case someone feels tempted to mess with these properties ;-)
```suggestion
  constructor(public readonly previousValue: T, public readonly currentValue: T, public readonly firstChange: boolean) {}
```

Reply:
done